CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+Contributions to `ros2_rust` are very welcome!
+
+Work is coordinated in the [Matrix chat](https://matrix.to/#/+rosorg-rust:matrix.org). It's a good idea to check in with the other contributors there before starting development on a new feature.
+
+There are also occasional ROS 2 Rust WG meetings that are announced on the [ROS discourse forum](https://discourse.ros.org/).
+
+## Coding guidelines
+Aim for well documented and tested code.
+
+Use declarative sentences, not imperative sentences, for documenting functions. For instance, `Creates a by-value iterator.`, not `Create a by-value iterator`. 
```

Comment:
```suggestion
For documenting functions, use declarative sentences, not imperative sentences. For instance, `Creates a by-value iterator.` should be used instead of `Create a by-value iterator`. 
```

Reply:
Done.